mix.exs

```diff
@@ -31,7 +31,7 @@ defmodule MongoosePush.Mixfile do
   defp deps do
     [
       {:chatterbox, github: "joedevivo/chatterbox", ref: "1f4ce4f", override: true},
-      {:sparrow, github: "esl/sparrow", ref: "b1896ca"},
+      {:sparrow, github: "esl/sparrow", ref: "80a17bd"},
```

Comment:
I'm only scared that this might clash with what has just been merged to master when removing :mock, this commit hash is older than that one.

Reply:
yes, we have to resolve that manually, but it's not a problem